katas/pokerhands/bryan/whoWins.js

```diff
@@ -0,0 +1,209 @@
+
+
+export class whoWins {
+    constructor(player1, player2) {
+        this.player1 = player1;
+        this.player2 = player2;
+
+
+        //return [player1,player2];
+    }
+
+    turnNum(strn) {
+        var nums = 0;
+        var comp = ['2', '3', '4', '5', '6', '7', '8', '9', 'T', 'J', 'Q', 'K', 'A'];
+        var sizec = comp.length;
+        for (var keyl1 = 0; keyl1 < sizec; keyl1++) {//turn them to numbers
+            if (strn == comp[keyl1]) {
+                nums = keyl1 + 2;
+            }
+        }
+        return nums;
+    }
+
+
+    whatSuit(wtag, wsuit) {
```

Comment:
This works, but the function is too big, as improvement, you should consider avoid having a lot of if-else blocks and create separated functions for each case that you find in the code, that way, testing could be also easier to do as you concentrate in smaller units rather than big functionality in one single function.